components/formats-gpl/src/loci/formats/in/OperettaReader.java

```diff
@@ -612,17 +721,22 @@ else if ("ImageResolutionY".equals(currentName)) {
         else if ("PositionX".equals(currentName)) {
           // position stored in meters
           final double meters = Double.parseDouble(value);
-          activePlane.positionX = new Length(meters, UNITS.METRE);
+          // see "OrientationMatrix" below
+          xPositionMeters = xPositionMeters == null ? meters : xPositionMeters * meters;
         }
         else if ("PositionY".equals(currentName)) {
           // position stored in meters
           final double meters = Double.parseDouble(value);
-          activePlane.positionY = new Length(meters, UNITS.METRE);
+          // see "OrientationMatrix" below
+          yPositionMeters = yPositionMeters == null ? meters : yPositionMeters * meters;
         }
-        else if ("AbsPositionZ".equals(currentName)) {
+        else if (("AbsPositionZ".equals(currentName) && !isHarmony) ||
```

Comment:
So datasets exported from Harmony contain both `PositionZ` and `AbsPositionZ` metadata. What is `AbsPositionZ` expected to refer to in this case? Will the value be conserved in the original metadata table?

Reply:
```PositionZ``` is relative to the bottom of the well.  ```AbsPositionZ``` is relative to whatever the default position of the ZDrive is, and may or may not be present or reliable.

None of the per-plane metadata is included in the original metadata table, since that can quickly add up to thousands of keys and result in import performance degradation.  If it would be helpful to add ```AbsPositionZ``` only though, that's probably fine.  I could also see a case for adding an option to toggle which key is used for backwards compatibility.